exercises/practice/accumulate/.meta/config.json

```diff
@@ -1,10 +1,35 @@
 {
   "blurb": "Implement the `accumulate` operation, which, given a collection and an operation to perform on each element of the collection, returns a new collection containing the result of applying that operation to each element of the input collection.",
-  "authors": [],
+  "authors": [
+    {
+      "github_username": "SaschaMann"
+    }
+  ],
+  "contributors": [
+    {
+      "github_username": "kytrinyx"
+    },
+    {
+      "github_username": "fsouza"
+    },
+    {
+      "github_username": "Teo-ShaoWei"
+    },
+    {
+      "github_username": "logankilpatrick"
+    },
+    {
+      "github_username": "guilhermebodin"
+    }
+  ],
```

Comment:
This is completely wrong, see https://github.com/exercism/julia/commits/v2-final-version/exercises/accumulate

Maybe because the exercise has been deprecated?